.travis.yml

```diff
@@ -1,21 +1,28 @@
 language: d
 
+os:
+  - linux
+  - osx
+
+# Order: oldest DMD, newest LDC, oldest LDC
+# Latest DMD is in the matrix.include to give it the `COVERAGE` env variable
+d:
+  - dmd-2.085.1
+  - ldc-1.19.0
+  - ldc-1.15.0
+  - dmd-nightly
+  - ldc-latest-ci
+
 matrix:
   include:
-    - d: dmd-beta
-    - d: dmd-2.088.0
+    - d: dmd-2.090.0
+      os: linux
       env: [COVERAGE=true]
-    - d: dmd-2.076.1
-    - d: ldc-1.17.0
-    - d: ldc-1.16.0
-    - d: ldc-1.15.0
-    - d: ldc-1.14.0
-    - d: ldc-1.6.0
-    - d: dmd-2.087.1
-    - d: dmd-2.086.1
-    - d: dmd-2.085.1
-    - d: dmd-2.084.1
```

Comment:
Could we keep at least a partial test for DMD 2.076.1 here (e.g. just `dub build` together with the suggested build type change)? Since GDC is stuck at that version (AFAIK), I'd like to keep basic support as long as it requires no serious extra effort.